frontend build도중 prettier 에러 발생

# What
```
$ npm run build

> frontend@0.1.0 build
> vue-cli-service build


/  Building for production...

 ERROR  Failed to compile with 2 errors                                                                                                                                                             오후 7:17:42

 error  in ./src/views/first/components/registmodal.vue

Module Error (from ./node_modules/eslint-loader/index.js):

C:\Users\S20154199\Desktop\CSPS\frontend\src\views\first\components\registmodal.vue
  71:26  error  Replace `␍⏎····················auth-nickname␍⏎····················register-btns␍⏎····················horizontal-r␍⏎····················disabled-check␍⏎··················` with `auth-nickname·register-btns·horizontal-r·disabled-check`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.


 @ ./node_modules/cache-loader/dist/cjs.js??ref--13-0!./node_modules/thread-loader/dist/cjs.js!./node_modules/babel-loader/lib!./node_modules/vue-cli-plugin-quasar/lib/loader.js.transform-quasar-imports.js!./node_modules/vue-cli-plugin-quasar/lib/loader.vue.auto-import-quasar.js??ref--1-0!./node_modules/cache-loader/dist/cjs.js??ref--1-1!./node_modules/vue-loader-v16/dist??ref--1-2!./src/views/first/main.vue?vue&type=script&lang=js 4:0-55 8:4-15
 @ ./src/views/first/main.vue?vue&type=script&lang=js
 @ ./src/views/first/main.vue
 @ ./src/router/index.js
 @ ./src/main.js
 @ multi ./src/main.js

 error  in ./src/views/information/problem-info/main.vue

Module Error (from ./node_modules/eslint-loader/index.js):

C:\Users\S20154199\Desktop\CSPS\frontend\src\views\information\problem-info\main.vue
  21:20  error  Replace `␍⏎··············column␍⏎··············justify-center␍⏎··············item-center␍⏎··············q-mt-lg␍⏎··············script-category·script␍⏎············` with `column·justify-center·item-center·q-mt-lg·script-category·script`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.


 @ ./src/router/index.js 37:23-75
 @ ./src/main.js
 @ multi ./src/main.js

 ERROR  Build failed with errors.
```

# Why
`class`명이 길어지면서 줄 바꿈 되어 prettier 설정과 맞지 않아서 발생하는 오류

# How
`class`명을 단일 라인으로 적기

--- a/frontend/src/views/first/components/registmodal.vue
+++ b/frontend/src/views/first/components/registmodal.vue
@@ -68,12 +68,7 @@
                 />
                 <div
                   @click="nickNameCheck"
-                  class="
-                    auth-nickname
-                    register-btns
-                    horizontal-r
-                    disabled-check
-                  "
+                  class="auth-nickname register-btns horizontal-r disabled-check"
                 >
                   중복확인
                 </div>

--- a/frontend/src/views/information/problem-info/main.vue
+++ b/frontend/src/views/information/problem-info/main.vue
@@ -18,13 +18,7 @@
             <div class="scroll-arrow"></div>
           </div>
           <div
-            class="
-              column
-              justify-center
-              item-center
-              q-mt-lg
-              script-category script
-            "
+            class="column justify-center item-center q-mt-lg script-category script"
           >
             <q-btn unelevated flat><b>Vue</b> </q-btn>
             <q-btn unelevated flat><b>React</b> </q-btn>

